cmd/version.go

```diff
@@ -32,7 +32,7 @@ var versionCmd = &cobra.Command{
 	Short: "Displays the current version",
 	Long:  `Displays the current version of the application`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Version: 0.2.3-pre-release")
+		fmt.Println("Version: 0.2.4")
```

Comment:
In Cobra, the version command should just be an attribute of the root `cobra.Command` [see example here](https://github.com/nathandines/forge/blob/master/commands/root.go#L26-L36)